doc/api/ssl.rst

```diff
@@ -598,6 +598,11 @@ Connection objects have the following methods:
     but not it returns the entire list in one go.
 
 
+.. py:method:: Connection.get_protocol_version()
+
+    Retrieve the version of the SSL or TLS protocol used by the Connection
```

Comment:
This should include an example of what the result looks like. Also should end with a `.` :-)